### DISCUSSION TITLE
Proc_open is required?

### DISCUSSION THREAD

**Discussion by artparesse** [spatie/laravel-backup#1603] [Q&A]
Proc_open is required? i mean, is there a way to use laravel-backup without proc_open? i can't enable it from security reason, so i need to find a solution to work package without that function.

PHP 8.1
Laravel 9
laravel-backup 8.1

**Comment by erikn69** [comment]
I couldn't find where it is being used, but feel free to make a PR to change it
https://github.com/spatie/laravel-backup/discussions/1122#discussioncomment-83662
